distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -1169,3 +1169,5 @@ type Number_Parse_Error
     to_display_text : Text
     to_display_text self =
         "Could not parse " + self.text.to_text + " as a double."
+
+Float.from (that:Integer) = (1.0 * that):Float
```

Comment:
Shouldn't we use some builtin that does the conversion instead on relying on semantics of the `*` node?

Reply:
But maybe it doesn't matter too much

Reply:
I was searching for some existing function to call. But I found only `floor` & co. No function that would work the opposite way (from `Integer` to `Float`)

Builtin would add fifty lines of code, multiplication by `1.0` is a one-liner. The performance shall be identical.

Reply:
Ok, fair